src/WebJobs.Extensions.EasyTables/Config/EasyTableConfiguration.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Configuration;
+using Microsoft.Azure.WebJobs.Host.Config;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EasyTables
+{
+    /// <summary>
+    /// Defines the configuration options for the EasyTable binding.
+    /// </summary>
+    public class EasyTableConfiguration : IExtensionConfigProvider
+    {
+        internal const string AzureWebJobsMobileAppUriName = "AzureWebJobsMobileAppUri";
+        internal const string AzureWebJobsMobileAppApiKeyName = "AzureWebJobsMobileAppApiKey";
+
+
+        /// <summary>
+        /// Constructs a new instance.
+        /// </summary>
+        /// <param name="resolver">The <see cref="INameResolver"/> to use for resolving the MobileAppUri.</param>
+        public EasyTableConfiguration(INameResolver resolver)
```

Comment:
I recommend just removing this for now since it just adds confusion - none of the other bindings to this, and it will just be dead code, waiting for a change that doesn't have a tracking item and may never happen.

If the desire is to make changes to the core name resolved, a bug can be opened for that, and we can discuss.